website/content/en/docs/building-operators/golang/advanced-topics.md

```diff
@@ -285,6 +285,70 @@ func main() {
 
+### Multiple architectures
+
+Operators may run on various architectures: x86_64, aarch64, ppc64le, s390x, etc.
+
+#### Manifest lists
+
+The most straightforward way of building operators and operands supporting multiple architectures is to leverage manifest lists, specified by [Image Manifest V2, Schema 2](https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list) or [OCI Image Index](https://github.com/opencontainers/image-spec/blob/main/image-index.md). A manifest list points to specific image manifests for one or more platforms.
+
+For convenience tools like [buildah](https://github.com/containers/buildah/blob/main/docs/buildah-bud.md#building-an-multi-architecture-image-using-a---manifest-option-requires-emulation-software) allow to cross-build and manifest multi-arch containers on one host. For instance:
+
+```shell
+for a in amd64 arm64 ppc64le s390x; do \
+  buildah bud --manifest registry/username/repo:v1 --arch $a; \
+done
+```
+
+This creates the manifest list, builds each image and adds them to the manifest list.
+
+The result can then be pushed to the desired registry.
+
+```shell
+buildah push registry/username/repo:v1
+```
+
+Docker with [buildx](https://docs.docker.com/buildx/working-with-buildx/#build-multi-platform-images) provides similar capabilities.
+
+```shell
+docker buildx build --push --platform linux/amd64,linux/arm64,linux/ppc64le,linux/s390x --tag registry/username/repo:v1 .
+```
+
+See [docker documentation](https://docs.docker.com/buildx/working-with-buildx/#build-multi-platform-images) for additional options.
+ 
+Note: the Dockerfile generated by the SDK for the operator explicitely references GOARCH=amd64 for go build. This can be amended to GOARCH=$TARGETPLATFORM. Docker will automatically set the environment variable to the value specified by --platform. With buildah --build-arg will need to be used for the purpose.
+
+All image references in the ClusterServiceVersion should be manifest lists containing the pointers to the image manifests for the supported architectures.
+
+#### Architecture specific content
+
+Operand images that require different image content (such as unique runtime packages) per architecture should be handled by the operator itself based on environment variables. That is, the CSV can list environment variables for each supported architecture / OS, for example:
+
+```
+MY_OPERAND_ARM64: my/arm64/image
+MY_OPERAND_PPC64LE: my/ppc64le/image
+```
+
+Based on the architecture and OS of the container the operator is running in, the operator can decide which environment variable to use to deploy the correct operands. When using a go-based operator the GOARCH environment variable in the container can provide the required information.
```

Comment:
This seems like it would only work it homogeneous clusters. What about clusters that have different nodes with different architectures where the operator may be running on `amd64` but the operand gets scheduled on `arm64`?

Reply:
It is a bit of a corner case. Multi-arch clusters are for instance not supported with OpenShift today.
For clusters that have this setup cluster-admins would need either:
- to use taints and toleration
- or to use NodeSelectors leveraging labels set by the Node Feature Discovery for instance
so that operators and operands run on the same architecture. This is however not driven by the operator authors.

The only (and preferred) approach that really works is with manifest lists as the container engine would pull the right image depending on the architecture.

I can add a caveat to highlight the scenario you are referring to.

Reply:
As discussed I removed the paragraph